bokehjs/test/models/plots/plot_canvas.coffee

```diff
@@ -25,11 +25,13 @@ Toolbar = utils.require("models/tools/toolbar").Model
 describe "PlotCanvas.Model", ->
 
   it "should set the responsive mode to box by default", ->
-    p = new PlotCanvas({x_range: new DataRange1d(), y_range: new DataRange1d()})
+    pp = new Plot({x_range: new DataRange1d(), y_range: new DataRange1d()})
+    p = pp.plot_canvas
```

Comment:
We're unit testing plot canvas here. PlotCanvas should be initiated and tested.

(Same for all below, or move the test)